# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/DocAccessible.cpp
# Commit: 50ec4059020a
# Full Hash: 50ec4059020abd34f64c60e59849e803e4adfbf6
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2019-09-18 09:56:40
# Regressor Bug: 1580292
# File Overlap Count: 1
# Description:
#   Bug 1580292 - Check for pruned descendants of reframed body. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45423
# ==============================================================================

diff -r 9aab8ce7d9f1 -r 50ec4059020a accessible/generic/DocAccessible.cpp
--- a/accessible/generic/DocAccessible.cpp	Thu Sep 12 01:00:15 2019 +0000
+++ b/accessible/generic/DocAccessible.cpp	Wed Sep 11 18:32:07 2019 +0000
@@ -1206,6 +1206,11 @@
   if (acc) return acc;
 
   acc = GetContainerAccessible(aNode);
+  if (acc == this && aNode == mContent) {
+    // The body node is the doc's content node.
+    return acc;
+  }
+
   if (acc) {
     uint32_t childCnt = acc->ChildCount();
     for (uint32_t idx = 0; idx < childCnt; idx++) {